### PR TITLE
feature: 717 performance section

### PR DESCRIPTION
**Pull request by delemaf**
### :pushpin: References

-   **Issue:** Closes #?

### CODE REVIEW

File: src/data/repositories/PerformanceOverviewD2Repository.ts [hunk 1]
```diff
@@ -204,6 +215,46 @@ export class PerformanceOverviewD2Repository implements PerformanceOverviewRepos
...
+                        type, // Set the valid type here with narrowed types
+                    },
+                ];
+            });
+        };
+
+        return apiToFuture(
+            this.api.analytics.get({
+                dimension: [
+                    `dx:${INDICATORS_717_PERFORMANCE.map(({ id }) => id).join(";")}`,
```
Comment (anagperal) [inline src/data/repositories/PerformanceOverviewD2Repository.ts:248]: In the future we can get these Program Indicators from DataStore like Maps.

File: src/data/repositories/PerformanceOverviewD2Repository.ts [hunk 2]
```diff
@@ -204,6 +215,46 @@ export class PerformanceOverviewD2Repository implements PerformanceOverviewRepos
...
+                    "pe:THIS_YEAR",
```
Comment (anagperal) [inline src/data/repositories/PerformanceOverviewD2Repository.ts:249]: Period dimension has to be this year?

File: src/webapp/pages/dashboard/use717Performance.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            {} as Record<string, typeof performanceIndicators>
+        );
+        return Object.entries(performanceIndicatorsByName).map(([key, values]) => {
+            const percentObj = values.find(item => item.type === "percent");
+            const countObj = values.find(item => item.type === "count");
+
+            const percent = percentObj ? percentObj.value : 0;
+            const count = countObj ? countObj.value : 0;
+
+            let color: "green" | "red" | "grey" | "normal" | undefined;
```
Comment (anagperal) [inline src/webapp/pages/dashboard/use717Performance.ts:43]: Maybe better to create a function to get the color

**Review by anagperal — CHANGES_REQUESTED** [review]
Thanks @9sneha-n ! Just a couple of comments (or now that Florian is not here should I chage it?)

File: src/data/repositories/PerformanceOverviewD2Repository.ts (hunk 2 above)
Comment (9sneha-n) [reply src/data/repositories/PerformanceOverviewD2Repository.ts:249]: checked with team.

**Review by anagperal — APPROVED** [review]
thanks @9sneha-n !!

**Pull request merged**